UnityProject/Assets/Scripts/Managers/NetworkManagement/CustomNetworkManager.cs

```diff
@@ -151,18 +151,24 @@ public void SteamServerStart()
 
 	}
 	
-
-// This method processes the callback data when authentication statuses change
-public void AuthChange(ulong steamid, ulong ownerid, ServerAuth.Status status)
-{
-	//TODO Add function to authentication and set persistent authentication flag on player.
-	bool Authed;
-	Authed = status == ServerAuth.Status.OK;
-
-	Debug.Log( "steamid: {0}" + steamid );
-	Debug.Log( "ownerid: {0}" + ownerid );
-	Debug.Log( "status: {0}" + status );
-}
+	/// Processes the callback data when authentication statuses change
+	public void AuthChange(ulong steamid, ulong ownerid, ServerAuth.Status status)
+	{
+		Debug.Log( $"steamid: {steamid}, ownerid: {ownerid}, status: {status}" );
```

Comment:
This here you should kick if Auth status != OK

Reply:
(because when VAC status is bad or PublisherBan has been issued, the player should also be kicked because auth will return a not-OK status)

Reply:
I take you the auth results for me to add?